src/app/components/privacy-policy/privacy-policy.component.scss

```diff
@@ -54,6 +54,7 @@
       right: 4%;
       height: 50px;
       cursor: pointer;
+      z-index: 4 !important;
```

Comment:
Why was this added? Decreasing the `z-index` of footer works!

Reply:
yeah but just to make sure that it's z-index remains high then others elements

Reply:
Then it should be high number like 100, right?